src/Makefile

```diff
@@ -24,7 +24,7 @@ LIB		=	-lpthread -lpolarssl -L. -L/usr/local/lib
 CHARDEN		=	-fstack-protector -fPIC -D_FORTIFY_SOURCE=2 -O1
 LHARDEN		=	-pie -fPIE
 WFLAGS		=	-Wall -Werror -Wextra
-CFLAGS		=	$(WFLAGS) $(DEFINES) $(INC) $(CHARDEN)
+CFLAGS		+=	$(WFLAGS) $(DEFINES) $(INC) $(CHARDEN)
```

Comment:
Doesn't it need an 'override' keyword, like for the LDFLAGS (l28)? Not sure how the CFLAGS variable is defined, maybe you don't need it, just tell me.